tests/receivers/envoy/testdata/otlp_exporter.yaml

```diff
@@ -0,0 +1,28 @@
+#receivers:
```

Comment:
If we change the template this can become out of date, any benefit in keeping it here?

Reply:
I'll remove.